api/models/container.go

```diff
@@ -27,6 +27,7 @@ const (
 
 	ImageBuilderComponentType     = "image_builder"
 	ModelComponentType            = "model"
+	PredictorComponentType        = "predictor"
```

Comment:
I'm adding a new predictor component type because Kserve is using `predictor` for their inference service component labels.

Reply:
Just curious, is it possible to remove `ModelComponentType` completely? (or replace/rename/refactor it with the new `PredictorComponentType`?) I think all the previous instances where we're using that constant is when they were referring to a predictor right?

Reply:
Yes, it's possible. For Merlin itself, it's mainly being used for Log UI. I have also considered this and I think it should be addressed in separate PR.

Reply:
Ahh I see, okay then! Thanks for clarifying!